rca/static_src/javascript/programmes/components/ProgrammesResults/ProgrammesResults.js

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { programmePageShape } from '../../programmes.types';
+
+import Icon from '../Icon/Icon';
+import ProgrammeTeaser from './ProgrammeTeaser';
+
+const getResultsCount = (count) => {
+    switch (count) {
+        case 0: {
+            return 'No results match your search';
+        }
+        case 1: {
+            return '1 result matches your search';
+        }
+        default: {
+            return `${count} results match your search`;
+        }
+    }
+};
+
+/**
+ * A list of programmes matching a search or filter.
+ * The list auto-magically appears when matches are found.
+ */
+const ProgrammesResults = ({ programmes, hasActiveSearch }) => {
+    const [activeProgramme, setActiveProgramme] = useState(null);
+    if (!hasActiveSearch) {
+        return null;
+    }
+
+    const count = getResultsCount(programmes.length);
+    return (
+        <>
+            <div className="programmes-results bg bg--dark section">
+                <div className="grid">
+                    <div className="programmes-results__actions">
+                        <button
+                            type="button"
+                            className="button programmes-results__back body body--one"
+                        >
+                            <Icon
+                                name="arrow"
+                                className="programmes-results__back__icon"
+                            />
+                            <span className="programmes-results__back__text">
+                                Back
+                            </span>
```

Comment:
I’ll add `onClick` to this in a future PR – for now this is just meant to be visually accurate.

Reply:
Ah cool, saw this and was going to ask